services/core-api/app/api/projects/project_summary/models/project_summary.py

```diff
@@ -315,6 +329,20 @@ def create_or_update_authorization(self, authorization):
                     ams_tracking_number=authorization.get('ams_tracking_number'),
                     ams_outcome=authorization.get('ams_outcome')
                 )
+# Check only for new files
```

Comment:
Another tiny nitpick, but would be nice to have this indented to the level of the fuction.